assets/model_monitoring/components/src/data_drift_compute_metrics/compute_data_drift.py

```diff
@@ -62,10 +64,14 @@ def compute_data_drift_measures_tests(
             categorical_columns_names,
             categorical_threshold,
         )
-
+        print(categorical_df)
     # TODO: fix this if, else
     if len(numerical_columns_names) != 0 and len(categorical_columns_names) != 0:
```

Comment:
In any case that len(categorical_columns_names) != 0 but categorical_df is None?

and is it possible that numerical_df is None?

Reply:
never faced this issue, if we ever encounter that we can do it. or do you think we should do it now, I can add it.